project-reports/2023/2023-Q3-Hyperledger-Cello.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger Cello
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+The Hyperledger Cello Project is in good health, with the recent release of version v1.0.0-RC, which includes essential dependency upgrades and bug fixes.
+
+v1.0.0-RC upgrades the Cello dashboard's Node.js version, supports Apple Chip Machines for local development, and refines the Readme documentation.
+    
+
+
+# Questions/Issues for the TOC
+
+There are currently no outstanding issues.
+
+# Releases
+
+** v1.0.0-RC2 was released in September 2023.
+** v1.0.0-RC1 was released in June 2023.
+** v1.0.0-GA was released in May 2023.
+** v1.0.0-beta1 was completed in November 2022.
```

Comment:
Not a big deal, but for proper markdown formatting you can use a single asterisk.